analytics/src/analytics/integrations/etldb/migrations/versions/0006_add_opportunity_tables.sql

```diff
@@ -0,0 +1,129 @@
+    expected_number_of_awards         BIGINT,
+    estimated_total_program_funding   BIGINT,
+    award_floor                       BIGINT,
+    award_ceiling                     BIGINT,
+    additional_info_url               TEXT,
+    additional_info_url_description   TEXT,
+    forecasted_post_date              DATE,
+    forecasted_close_date             DATE,
+    forecasted_close_date_description TEXT,
+    forecasted_award_date             DATE,
+    forecasted_project_start_date     DATE,
+    fiscal_year                       INTEGER,
+    revision_number                   INTEGER,
+    modification_comments             TEXT,
+    funding_category_description      TEXT,
+    applicant_eligibility_description TEXT,
+    agency_code                       TEXT,
+    agency_name                       TEXT,
+    agency_phone_number               TEXT,
+    agency_contact_description        TEXT,
+    agency_email_address              TEXT,
+    agency_email_address_description  TEXT,
+    is_deleted                        BOOLEAN,
+    can_send_mail                     BOOLEAN,
+    publisher_profile_id              BIGINT,
+    publisher_user_id                 TEXT,
+    updated_by                        TEXT,
+    created_by                        TEXT,
+    created_at                        TIMESTAMP WITH TIME ZONE   NOT NULL,
+    updated_at                        TIMESTAMP WITH TIME ZONE   NOT NULL,
+    version_number                    INTEGER,
+    CONSTRAINT opportunity_summary_is_forecast_uniq
+        UNIQUE (is_forecast, revision_number, opportunity_id)
+);
+
+CREATE INDEX IF NOT EXISTS opportunity_summary_opportunity_id_idx
+    ON opportunity_summary (opportunity_id);
+
+CREATE TABLE IF NOT EXISTS  current_opportunity_summary
+(
+    opportunity_id         BIGINT                                 NOT NULL
+        CONSTRAINT current_opportunity_summary_opportunity_id_opportunity_fkey
+            REFERENCES  opportunity,
+    opportunity_summary_id BIGINT                                 NOT NULL
+        CONSTRAINT current_opportunity_summary_opportunity_summary_id_oppo_8251
+            REFERENCES  opportunity_summary,
+    opportunity_status_id  INTEGER                                NOT NULL
+        CONSTRAINT current_opportunity_summary_opportunity_status_id_lk_op_3147
+            REFERENCES  lk_opportunity_status,
+    created_at             TIMESTAMP WITH TIME ZONE   NOT NULL,
+    updated_at             TIMESTAMP WITH TIME ZONE   NOT NULL,
+    PRIMARY KEY (opportunity_id, opportunity_summary_id)
+);
+
+CREATE INDEX IF NOT EXISTS current_opportunity_summary_opportunity_id_idx
+    ON  current_opportunity_summary (opportunity_id);
+
+CREATE INDEX IF NOT EXISTS current_opportunity_summary_opportunity_status_id_idx
+    ON  current_opportunity_summary (opportunity_status_id);
+
+CREATE INDEX IF NOT EXISTS current_opportunity_summary_opportunity_summary_id_idx
+    ON  current_opportunity_summary (opportunity_summary_id);
```

Comment:
Looks like this one was copied twice

```suggestion
CREATE INDEX IF NOT EXISTS current_opportunity_summary_opportunity_status_id_idx
    ON  current_opportunity_summary (opportunity_status_id);
```

Reply:
they look similar but one is for `...status_id_idx` and the other is for `...summary_id_idx`

Reply:
Bah, I reread that 3 times and still missed it.